libraries/botbuilder-dialogs/src/memory/scopes/settingsMemoryScope.ts

```diff
@@ -8,6 +8,31 @@
 import { MemoryScope } from './memoryScope';
 import { ScopePath } from '../scopePath';
 import { DialogContext } from '../../dialogContext';
+import { DialogTurnStateConstants } from '../../dialogTurnStateConstants';
+
+/**
+ * The setting node.
+ */
+class Node {
+    /**
+     * Initializes a new instance of `Node`.
+     * @param value Value of the node. If the node is not leaf, value represents the current path.
```

Comment:
Our JSDoc should include the type, so this should be `@param {string} value ...`